diff --git a/dist/utils/validation.js b/dist/utils/validation.js
index 7b2b2164b9aa9e4363f1449883af670bd33efea1..9db98a2f5d32ca3107bbe7e540085f49ef36e434 100644
--- a/dist/utils/validation.js
+++ b/dist/utils/validation.js
@@ -31,7 +31,7 @@ export const parseInput = (input) => {
     const normalisedName = isValid ? nameReference : undefined;
     const labels = nameReference.split('.');
     const tld = labels[labels.length - 1];
-    const isETH = tld === 'eth';
+    const isETH = tld === 'etn';
     const labelDataArray = split(nameReference);
     const isShort = (labelDataArray[0].output?.length || 0) < MINIMUM_DOT_ETH_CHARS;
     if (labels.length === 1) {
@@ -56,5 +56,5 @@ export const parseInput = (input) => {
         labelDataArray,
     };
 };
-export const checkIsDotEth = (labels) => labels.length === 2 && labels[1] === 'eth';
+export const checkIsDotEth = (labels) => labels.length === 2 && labels[1] === 'etn';
 //# sourceMappingURL=validation.js.map
